style.css

```diff
@@ -1,37 +1,61 @@
 body {
   font-family: 'Poppins', sans-serif;
   margin: 0;
+  background-color: #f7f7f9;
+}
+
+header {
+  background-color: #fff;
 }
 
 nav {
   font-weight: 700;
   display: flex;
   justify-content: space-between;
+  align-items: center;
   color: #6070ff;
   font-size: small;
-  margin-top: 4%;
 }
 
 .intro {
   height: 100vh;
   background-image: url("images/header-shapes\ mobile@2x.png");
   background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
   display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.

Reply:
I'm going to do my best into incorporating this to my code. 🙌💯

Reply:
Issue addressed, I refactored the code! 👨‍💻